Applications/MNIST/jni/main.cpp

```diff
@@ -334,7 +334,6 @@ int main(int argc, char *argv[]) {
     model->train();
     training_loss = model->getTrainingLoss();
     validation_loss = model->getValidationLoss();
-    last_batch_loss = model->getLoss();
```

Comment:
Q. is getLoss gone?

Reply:
`getLoss` returns the loss for the last batch, which by definition is strange.
If certain training characteristics are required, the `metrics` struct should be returned with all the characteristics.

Reply:
Yes. Sounds reasonable.